src/i18n/strings/en_EN.json

```diff
@@ -912,7 +915,6 @@
     "You've previously used a newer version of Riot on %(host)s. To use this version again with end to end encryption, you will need to sign out and back in again. ": "You've previously used a newer version of Riot on %(host)s. To use this version again with end to end encryption, you will need to sign out and back in again. ",
     "Incompatible Database": "Incompatible Database",
     "Continue With Encryption Disabled": "Continue With Encryption Disabled",
-    "Failed to indicate account erasure": "Failed to indicate account erasure",
```

Comment:
This string is no longer used after 328d57f06320d839992a6f4eff0fae0eac4c5dfe.

Reply:
also removed in https://github.com/matrix-org/matrix-react-sdk/pull/2338#discussion_r239960432 :p

(we should probably just commit it directly)